ui/src/data/source.tsx

```diff
@@ -852,7 +852,19 @@ function generateOccurrenceFilter(
         entity.entity,
       ];
     }
-    return [null, ""];
+
+    return [
```

Comment:
No, this is changing criteria with no values selected from a no-op to matching any instance of the criteria (e.g. any condition).

Reply:
I see, so that's basically like saying person has >=1 condition_occurrence for any condition? That sort of sounds like a modifier.

Reply:
It's "Any person with a condition_occurrence where condition = X" vs "Any person with a condition_occurrence". It just leaves off the where clause when it was previous being treated as a no-op. This is the default state for a condition criteria which has been added without actually selecting a condition rather than an intentional choice.